doc/source/reference/arrays.dtypes.rst

```diff
@@ -128,7 +128,7 @@ What can be converted to a data-type object is described below:
 
    Used as-is.
 
-:const:`None`
+`None`
```

Comment:
I think you want double backticks or no backticks for all of these. Not sure if this turns it into a link, but that's weird anyway.

Reply:
Here is the [source](https://raw.githubusercontent.com/python/cpython/master/Doc/library/constants.rst) as to why `:const:` is wrong - it should be `:data:`. In any case, sphinx makes this a link with no directive needed

Reply:
To be clear, what I'm saying is: the normal thing to do is to not make `None` a link at all.

Reply:
```
git grep "\`None\`"
```

turns up many more cases of this. Should the default be not to link?

Reply:
Yeah I'd say so. I remember a discussion about this when we first drew up the docstring standard.

Reply:
done